src/main/java/com/neuvector/Scanner.java

```diff
@@ -258,7 +266,11 @@ private static String runCMD(String[] cmdArgs){
 
     }
 
-    private static ScanRepoReportData runScan(String[] cmdArgs, String scanReportPath, String[] credentials){
+    private static ScanRepoReportData runScan(String[] cmdArgs, String scanReportPath, String[] credentials) {
+        if (isRootFile(getScanReportPath(scanReportPath))) {
+            //we need to clean the empty args
```

Comment:
I do not think we need this..

Reply:
if not cmdArguments will look like this `["docker", "run", "", "", "--name", +19 more]` and then in the runScan method we will have an error like:  `invalid reference format.See 'docker run --help'.` that's why I am cleaning the empty Strings when the file is root, added in the method:

```
static String[] getDockerGroupCmdArgs(String scanReportPath) {
        String[] cmdGroupArgs = {"", ""};
        if (!scanResultsFileExist(scanReportPath) || !isRootFile(scanReportPath)) {
            cmdGroupArgs[0] = "-u";
            cmdGroupArgs[1] = executeCommand("id -u");
            return cmdGroupArgs;
        }
        return cmdGroupArgs;
    }
```